src/qt3utils/datagenerators/piezoscanner.py

```diff
@@ -45,8 +45,6 @@ def set_to_starting_position(self):
     def close(self):
         self.rate_counter.close()
 
-    def set_num_data_samples_per_batch(self, N):
-        self.rate_counter.num_data_samples_per_batch = N
```

Comment:
@gadamc ^

Reply:
These were removed because they were no longer needed. It's a little outside the scope of this work, but it was part of the refactoring that was necessary to implement an interface class for spectrometers in qt3scan.